helix-view/src/editor.rs

```diff
@@ -70,6 +73,10 @@ pub struct Editor {
     pub registers: Registers,
     pub theme: Theme,
     pub language_servers: helix_lsp::Registry,
+
+    pub debugger: Option<helix_dap::Client>,
+    pub debugger_events: SelectAll<UnboundedReceiverStream<helix_dap::Payload>>,
```

Comment:
Sidenote: I've been wanting to make this simply `UnboundedReceiverStream` since there's always just a single debugger (no need for SelectAll), we just need to use an empty receiver stream if no debugger)

Reply:
We can. Or better leave space for multiple debuggers (definitely not for this PR)